libdevcore/SourceLocation.h

```diff
@@ -28,18 +28,23 @@
 
 namespace dev
 {
-namespace solidity
-{
 
 /**
  * Representation of an interval of source positions.
  * The interval includes start and excludes end.
  */
-struct Location
+struct SourceLocation
 {
-	Location(int _start, int _end, std::shared_ptr<std::string const> _sourceName):
+	SourceLocation(int _start, int _end, std::shared_ptr<std::string const> _sourceName):
 		start(_start), end(_end), sourceName(_sourceName) { }
-	Location(): start(-1), end(-1) { }
+	SourceLocation(): start(-1), end(-1) { }
```

Comment:
Could we get rid of all constructors by initializing start and end at the point of their declaration?